File: commands/fun/pp.js
module.exports = {
    name: 'pp',
    aliases: [],
    permissions: [],
    cooldown: 0,
    async execute(client, message, cmd, args, Discord) {

        const randomnumber = Math.floor(Math.random() * (50 - 2) + 2);
        const alphabet = '='
        const author = message.mentions.users.first() || message.author
        const ppsize = alphabet.repeat(Math.floor(Math.random() * randomnumber))

        const ppcalc = new Discord.MessageEmbed()
            .setColor('RANDOM')
            .setTitle(`peepee size machine`).setDescription(`${author}'s pp \n 8${ppsize}D`)
        message.channel.send(ppcalc);

    }
}

File: commands/Fun/pp.py
from setting import *

@client.command()
async def pp(ctx, member:discord.Member=None):
    
    # Variable
    random_number = randint(1, 50)
    alphabet = '='
    ppsize = alphabet * random_number
    member = member or ctx.author
    embed = discord.Embed(title='PP size machine', description=f"{member.mention}'s pp \n 8{ppsize}D", color=0x00f2ff)

    await ctx.send(embed=embed)

# Error(s)
@pp.error
async def pp_error(ctx, error):
    
    # Unknown error
    await unknown_error(ctx, error)
